resources/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Resource(models.Model):
+
+    name = models.CharField(max_length=254)
+    link = models.URLField(max_length = 200) 
```

Comment:
Do we only need these keys? I would suggest maybe to add a description as well to give an indication of what you would use each resource for.

Reply:
Hi @stefdworschak ! Would you suggest to make a description field required as well as name and link? Or should it be optional?